script/release/publish-to-npm.js

```diff
@@ -127,10 +128,10 @@ new Promise((resolve, reject) => {
         JSON.stringify(currentJson, null, 2)
       );
     } else {
-      if (currentBranch === 'master') {
-        // This should never happen, master releases should be nightly releases
+      if (currentBranch === 'master' || currentBranch === 'main') {
+        // This should never happen, main releases should be nightly releases
         // this is here just-in-case
-        npmTag = 'master';
+        npmTag = 'main';
```

Comment:
Hm this is a tricky one, as the comment says this should never happen.  In fact `npm show electron@master` and `npm show electron-nightly@master` don't show anything.  We should probably take this opportunity to put a `throw` here instead 😆

Reply:
happy to take your ```suggestion, my aim here is to do as little surgery as possible for this particular refactor.